curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b0.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-5
 
 # --description--
 
-Now, you should check if the applicant is qualified for a car loan only. If they're, return the string `"You qualify for a car loan."`.
+Now, you should check if the applicant is qualified for a car loan only. If they are qualified, return the string `"You qualify for a car loan."`.
```

Comment:
when you create a branch, and then a PR, make sure you do not have extra commits in it. Like this change is unrelated.